<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>overlay_style_picker</class>
 <widget class="QDialog" name="overlay_style_picker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>212</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="5" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="material_name">
     <property name="text">
      <string>MaterialName</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="ring_style_group">
     <property name="title">
      <string>Ring Style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QLabel" name="ring_linestyle_label">
        <property name="text">
         <string>Line Style:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ring_color_label">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ring_color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="ring_linestyle">
        <item>
         <property name="text">
          <string>solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dotted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashdot</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="ring_linewidth_label">
        <property name="text">
         <string>Line Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QDoubleSpinBox" name="ring_linewidth">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="range_style_group">
     <property name="title">
      <string>Range Style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="range_color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="range_color_label">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="range_linestyle_label">
        <property name="text">
         <string>Line Style:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="range_linestyle">
        <property name="currentText">
         <string>solid</string>
        </property>
        <item>
         <property name="text">
          <string>solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dotted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashdot</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="range_linewidth_label">
        <property name="text">
         <string>Line Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QDoubleSpinBox" name="range_linewidth">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>overlay_style_picker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>overlay_style_picker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
